packages/beacon-node/test/perf/chain/opPools/opPool.test.ts

```diff
@@ -0,0 +1,110 @@
+import {itBench} from "@dapplion/benchmark";
+import {
+  MAX_ATTESTER_SLASHINGS,
+  MAX_BLS_TO_EXECUTION_CHANGES,
+  MAX_PROPOSER_SLASHINGS,
+  MAX_VOLUNTARY_EXITS,
+} from "@lodestar/params";
+import {CachedBeaconStateAltair} from "@lodestar/state-transition";
+import {ssz} from "@lodestar/types";
+// eslint-disable-next-line import/no-relative-packages
+import {generatePerfTestCachedStateAltair} from "../../../../../state-transition/test/perf/util.js";
+import {OpPool} from "../../../../src/chain/opPools/opPool.js";
+import {generateBlsToExecutionChanges} from "../../../fixtures/capella.js";
+import {
+  generateIndexedAttestations,
+  generateSignedBeaconBlockHeader,
+  generateVoluntaryExits,
+} from "../../../fixtures/phase0.js";
+import {BlockType} from "../../../../src/chain/interface.js";
+
+// Aug 11 2021
+// getAttestationsForBlock
+//     ✓ getAttestationsForBlock                                             4.410948 ops/s    226.7086 ms/op        -         64 runs   51.8 s
+describe("opPool", () => {
+  let originalState: CachedBeaconStateAltair;
+
+  before(function () {
+    this.timeout(2 * 60 * 1000); // Generating the states for the first time is very slow
+
+    originalState = generatePerfTestCachedStateAltair({goBackOneSlot: true});
+  });
+
+  itBench({
+    id: "getSlashingsAndExits - default max",
+    beforeEach: () => {
+      const pool = new OpPool();
+      fillAttesterSlashing(pool, originalState, MAX_ATTESTER_SLASHINGS);
+      fillProposerSlashing(pool, originalState, MAX_PROPOSER_SLASHINGS);
+      fillVoluntaryExits(pool, originalState, MAX_VOLUNTARY_EXITS);
+      fillBlsToExecutionChanges(pool, originalState, MAX_BLS_TO_EXECUTION_CHANGES);
+
+      return pool;
+    },
+    fn: (pool) => {
+      pool.getSlashingsAndExits(originalState, BlockType.Full, null);
```

Comment:
if this does not modify oppool and the time tracked is too small, consider using `runsFactor` to make it more precise

Reply:
It's less than 100ms which is fine to measure.